src/features/common/Layout/Footer/SelectLanguageAndCountry.tsx

```diff
@@ -136,7 +129,10 @@ export default function TransitionsModal(props) {
 
 // Maps the radio buttons for countries
 function MapCountry(props) {
-  const { sortedCountriesData, value, handleChange } = props;
+  const { t } = useTranslation(['country']);
+  
+  const { value, handleChange } = props;
+  const sortedCountriesData = sortCountriesByTranslation(t);
```

Comment:
This add some unnecessary sorting to the app at runtime, but I cannot do the sorting on the translated country names before i18n is initialized.